superset/migrations/versions/811494c0cc23_remove_path_from_logs.py

```diff
@@ -0,0 +1,46 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+"""Remove path, path_no_int, and ref from logs
+
+Revision ID: 811494c0cc23
+Revises: 8ee129739cf9
+Create Date: 2020-12-03 16:21:06.771684
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = "811494c0cc23"
+down_revision = "8ee129739cf9"
+
+import sqlalchemy as sa
+from alembic import op
+
+from superset.migrations.shared import utils
+
+
+def upgrade():
+    with op.batch_alter_table("logs") as batch_op:
+        if utils.table_has_column("logs", "path"):
+            batch_op.drop_column("path")
```

Comment:
@etr2460 this isn't quite right. If the column exists one should really iterate over the non-NULL values and migrate those to the `json` column. That could be somewhat of an expensive operation but given the relevancy (or lack there of) and recency of the original migration it may be ok to accept the data loss.

Reply:
yeah, i made the executive decision to drop any extra metadata that came in in the last week from the previous change. I think it's reasonable since i doubt many folks are relying on them thus far

Reply:
I think it's ok to drop the metadata